components/Backgrounds/Background.styled.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export const TrackerFormBackground = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: start;
+  gap: 1rem;
+  padding-top: 1rem;
+  height: 100vh;
```

Comment:
I would consider changing this to `min-height`. That way, if the content of the page is larger the an the view-height, the page will grow (rather than being restricted at `100vh`;